templates/test/ci/prow-topology/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I think we should aim to get rid of these kustomize files as we move the templates to ClusterClass. Having both Kustomize and variables/patches in the topology will be too complex. Let's: 1) put the AzureClusterIdentity in the base cluster class (we want that for every cluster now) 2) move aim to move all addons including CSI proxy and CNI to Helm instead of CRS once #2264 merges.

Reply:
Ack, will wait for 2264 to merge.

Reply:
done.